### PR TITLE
(REFACTOR & UPDATE) Translations (German) mainly Notifications

### PR DESCRIPTION
**Pull request by Alkl58**
Greetings,

this is going to take a while to go through.

There are probably mistakes, so don't merge it as we discussed.

It will probably take a few days / weeks.

### CODE REVIEW

**Comment by Globbered** [comment]
Hi there @Alkl58 

was testing the controller change out you have added.
Tried:
`->withSuccess(trans('toasts.settings-profile-privacy'));`
and added
`'settings-profile-privacy'     => 'Your Profile Privacy Settings Have Been Saved!',`
to then en language folder and the language I need this for.

But I end up with:
Error 500: Internal Server Error!
Our server encountered an internal error.
Sorry For The Inconvenience

Once I click the button.

Is any other modification needed to make the system understand the syntax
`trans('toasts.settings-profile-privacy')` somewhere?

/Globber

**Comment by HDVinnie** [comment]
> Hi there @Alkl58
> 
> was testing the controller change out you have added.
> Tried:
> `->withSuccess(trans('toasts.settings-profile-privacy'));`
> and added
> `'settings-profile-privacy' => 'Your Profile Privacy Settings Have Been Saved!',`
> to then en language folder and the language I need this for.
> 
> But I end up with:
> Error 500: Internal Server Error!
> Our server encountered an internal error.
> Sorry For The Inconvenience
> 
> Once I click the button.
> 
> Is any other modification needed to make the system understand the syntax
> `trans('toasts.settings-profile-privacy')` somewhere?
> 
> /Globber


Might be useful if you said what the error 500 was for....

**Comment by Globbered** [comment]
Hi found the issue on my side.

@Alkl58 I am starting at the bottum of the controller listing then.
I have created all string translations to controllers.php file in the English language.

/Globber

**Comment by Alkl58** [comment]
I will close the PR, due to it being just "too big", which is hard to test and probably creates more problems than it solves.

In the future I will make smaller PRs, which are way more easy to handle than this monstrosity ^^

File: resources/lang/de/auth.php [hunk 1]
```diff
@@ -17,48 +17,50 @@
...
+    'allow-invite'           => 'Offene Registrierung ist aktuell geschlossen! Du musst eingeladen werden, um dich registrieren zu können! Du wurdest zur Anmeldeseite weitergeleitet!',
     'are-you'                => 'Bist du:',
     'application'            => 'Bewerbung',
     'appl-closed'            => 'Bewerbungen sind geschlossen',
-    'appl-intro'             => 'ist eine geschlossene Community. Du brauchst einen Invitelink, um dich registrieren zu können! Wenn du keinen Invite bekommen kannst, dann kannst du folgendes Bewerbungschreiben ausfüllen.',
+    'appl-intro'             => 'ist eine geschlossene Community. Du brauchst einen Invitelink, um dich registrieren zu können! Wenn du keinen Invite bekommen kannst, dann kannst du folgendes Bewerbungsschreiben ausfüllen.',
     'appl-reason'            => 'Wie hast du von ":sitename" erfahren? Wieso willst du reinkommen? ',
     'application-submitted'  => 'Deine Bewerbung wurde abgeschickt. Du wirst demnächst eine E-Mail erhalten!',
-    'banned'                 => 'Dieser Account wurde gebannt!',
+    'banned'                 => 'Dieser Nutzerkonto wurde gebannt!',
```
Comment (Luxa-Gaming) [inline resources/lang/de/auth.php:32]: 'Dieses Nutzerkonto wurde gebannt!'

File: resources/lang/de/user.php [hunk 2]
```diff
@@ -115,22 +129,34 @@
...
     'id-permissions'                       => 'ID & Rechte',
     'image'                                => 'Bild',
+    'image-deleted'                        => 'Bild wurde erfolgreich gelöscht',
+    'image-not-found'                      => 'Bild nicht gefunden! Bitte reporte dies dem Team!',
+    'image-success'                        => 'Dein Bild wurde erfolgreich veröffentlicht!',
     'important'                            => 'Wichtig',
     'important-info'                       => 'Wichtige Info',
     'information'                          => 'Information',
-    'invited-by'                           => 'Invited von',
+    'invitations-disabled'                 => 'Invitationen sind deaktiviert, da "Offene Registrierung" aktiv ist!',
```
Comment (Luxa-Gaming) [inline resources/lang/de/user.php:147]: 'Invites sind deaktiviert, da "Offene Registrierung" aktiv ist!'

File: resources/lang/de/user.php [hunk 3]
```diff
@@ -233,14 +282,14 @@
     'requested'                            => 'Requestes',
```
Comment (Luxa-Gaming) [inline resources/lang/de/user.php:282]: 'Requested'

**Review by Luxa-Gaming — COMMENTED** [review]
So, dann schau mal. Am besten natürlich selber nochmal nachsehen, ob das so passt.

**Pull request closed without merging**